vimeo-networking/src/main/java/com/vimeo/networking/model/Subscriptions.java

```diff
@@ -0,0 +1,108 @@
+package com.vimeo.networking.model;
+
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A model that holds the type of push subscriptions a user has. This class
+ * has both getters and setters since a user can update their subscriptions
+ * using a Patch.
+ * <p>
+ * Created by zetterstromk on 12/15/16.
+ */
+public class Subscriptions implements Serializable {
+
+    private static final long serialVersionUID = 3088065484753327987L;
+
+    public static final String KEY_COMMENT = "comment";
```

Comment:
So these are fixed as opposed to dynamic key/value pairs? How are they going to be changed going forward?